docs/content/references/cli.mdx

```diff
@@ -9,7 +9,7 @@ Sui provides a command line interface (CLI) tool to interact with the Sui networ
 
 ## Update CLI
 
-To get the latest version of the CLI, you can run the following command from a terminal or console. Be sure to replace `<BRANCH-NAME>` with `main`, `devnet`, `testnet`, or `mainnet` to get the desired version. For more information on the branches available, see [Sui Environment Setup](/guides/developer/getting-started/sui-environment.mdx).
+To get the latest version of the CLI, you can run the following command from a terminal or console. Be sure to replace `<BRANCH-NAME>` with `main`, `devnet`, `testnet`, or `mainnet` to get the desired version. For more information on the branches available, see [Sui Environment Setup](./contribute/sui-environment.mdx).
 
 ```shell
 cargo install --locked --git https://github.com/MystenLabs/sui.git --branch <BRANCH-NAME> --features gas-profiler sui
```

Comment:
Should we add a comment here that `--features gas-profiler` is only needed if you want to run a gas profile for a tx?